west.yml

```diff
@@ -193,7 +193,7 @@ manifest:
       groups:
         - hal
     - name: hal_nxp
-      revision: 1ed023da7f5541b78099b5a030507a2839ce554f
+      revision: pull/297/head
```

Comment:
the PR in HAL has been merged, please update the revision

Reply:
Now its PR 306